registry.json

```diff
@@ -5547,5 +5547,19 @@
    "telegram": "THEBMOREJOKER",
    "operatorName": "Lee",
    "github": "THEBMOREJOKER"
+  },
+  {
+      "ticker": "KAPY",
+      "poolId": "3adfc2b13579d2b924a668655f21aeb641190a3201217e6f7e4b169b",
+      "social": {
+          "website": "https://pool.kapybara.io",
+          "twitter": "kapypool",
```

Comment:
0 following 0 followers 1 post.

Come on make some effort, you need to get out there to get some delegators

Reply:
yeah... I was hoping that I can get some exposure by joining this group?